src/aero_reps/aero_rep_single_particle.c

```diff
@@ -437,8 +437,9 @@ void aero_rep_single_particle_print(int *aero_rep_int_data,
   printf("\nAerosol representation id: %d", AERO_REP_ID_);
   printf("\nMax computational particles: %d", MAX_PARTICLES_);
   printf("\nParticle state size: %d", PARTICLE_STATE_SIZE_);
-  printf("\n\n   - Phases -");
   for(int i_layer = 0; i_layer < NUM_LAYERS_; ++i_layer){
+    printf("\nLayer: %d", i_layer);
+    printf("\n\n   - Phases -");
```

Comment:
please remove debug output

Reply:
We added this to the print function to print layer properties. This should have been part of the last PR.